csrc/type.cpp

```diff
@@ -795,8 +795,6 @@ static const char* iter_type2string(IterType t) {
       return "r";
     case IterType::Broadcast:
       return "b";
-    case IterType::Gather:
-      return "g";
     case IterType::Stride:
```

Comment:
Is `IterType::Stride` still used?

Reply:
Apparently, no. I think we used to have some conv-like tests by using that, like strided pooling. Stride is fairly simple, so I think we could just leave it there for now, but I'd be ok to remove it as well.